sources/PPU/PPU.cpp

```diff
@@ -223,7 +445,7 @@ namespace ComSquare::PPU
 		return 0x3F;
 	}
 
-	uint16_t PPU::getVramAddress()
+	uint16_t PPU::getVramAddress() const
```

Comment:
This method should still be const.

Reply:
I don't get it the pr is adding the const